test/spec/general.js

```diff
@@ -87,7 +87,7 @@ define(function(require) {
 
       setupVerifyIdTokenTest({
         title: 'verifies a valid idToken',
-        idToken: tokens.standardIdToken,
+        idToken: tokens.verifiableIdToken,
```

Comment:
I found out the verify tests were failing after running the tests in Chrome (phantom doesn't have the crypto apis, so the tests don't run).